dashboard/android/key.properties

```diff
@@ -0,0 +1,4 @@
+storePassword=dartdart
```

Comment:
AFAIK this isn't an actual password. It just verifies you're using the expected key file.

Reply:
I think you're not supposed to check in key.properties? (and this may be the actual password?)

Reply:
Ah right, this is the password to the key file, which contains the actual password...